newIDE/app/src/UI/Menu/ContextMenu.js

```diff
@@ -58,7 +60,8 @@ const MaterialUIContextMenu = React.forwardRef<
           {...menuImplementation.getMenuProps()}
         >
           {menuImplementation.buildFromTemplate(
-            props.buildMenuTemplate(i18n, buildOptions)
+            props.buildMenuTemplate(i18n, buildOptions),
+            forceUpdate
```

Comment:
Add in `buildFromTemplate` in `ElectronMenuImplementation` the parameter and explain that it's not used.